src/modules/task/dto/get-task.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { Type } from 'class-transformer';
+import { IsNumber, IsOptional } from 'class-validator';
+
+export enum TaskSortByEnum {
+  id = 'id',
+  title = 'title',
+  description = 'description',
+}
+
+export enum TaskSortDirection {
+  asc = 'asc',
+  desc = 'desc',
+}
+
+export class PaginationDto {
+  @IsNumber()
+  @Type(() => Number)
+  @IsOptional()
+  limit: number;
```

Comment:
Давай значение по умолчанию проставим, чтобы если пользователь ничего не передал, всё равно работала пагинация
```suggestion
  limit: number = 10;
```